docs/source/features.rst

```diff
@@ -16,30 +16,900 @@ Text
+See details in :class:`~tensorbay.label.supports.CategoryInfo`.
+
+AttributeInfo
+--------------
+
+Each :ref:`attribute<features:Attributes>` of a label
+appeared in the dataset should follow the rules set in the attributes of the subcatalog.
+
+You can add attribute information to the subcatalog.
+
+.. code:: python
+
+    >>> box2d_subcatalog.add_attribute(
+    ... name="attribute_name",
+    ... type_="number",
+    ... maximum=100,
+    ... minimum=0,
+    ... description="attribute description"
+    ... )
+    >>> box2d_subcatalog.categories
+    NameOrderedDict {
+      'attribute_name': AttributeInfo("attribute_name")(...)
+    }
+
+We use :class:`~tensorbay.label.attributes.AttributeInfo` to describe the rules of an
+:ref:`attribute<features:Attributes>`, which refers to the `Json schema`_ method.
+
+See details in :class:`~tensorbay.label.attributes.AttributeInfo`.
+
+.. _Json schema: https://json-schema.org/
+
+Other unique subcatalog features will be explained in the corresponding label type section.
+
+Classification
+==============
+
+Classification is to classify data into different categories.
+
+It is the annotation for the entire file,
+so each data can only be assigned with one classification label.
+
+Classification labels applies to different types of data, such as images and texts.
+
+The structure of one classification label is like::
+
+        {
+            "category": <str>
+            "attributes": {
+                <key>: <value>
+                ...
+                ...
+            }
+        }
+
+
+
+To create a :class:`~tensorbay.label.label.Classification` label:
+
+.. code:: python
+
+    >>> from tensorbay.label import Classification
+    >>> classification_label = Classification(
+    >>> category="data_category",
+    >>> attributes={"attribute_name": "attribute_value"}
+    >>> )
+    >>> classification_label
+    Classification(
+      (category): 'data_category',
+      (attributes): {...}
+    )
+
+
+Classification.Category
+-----------------------
+
+The category of the entire data file.
+See :ref:`features:Category` for details.
+
+Classification.Attributes
+-------------------------
+
+The attributes of the entire data file.
+See :ref:`features:Attributes` for details.
+
+.. note::
+
+   Both category and attributes are not required in one classification label,
+   but one of them must exist.
+
+ClassificationSubcatalog
+------------------------
+
+Before adding the classification label to data,
+:class:`~tensorbay.label.subcatalog.ClassificationSubcatalog` should be defined.
+
+:class:`~tensorbay.label.subcatalog.ClassificationSubcatalog`
+has categories and attributes information,
+see :ref:`features:CategoryInfo` and :ref:`features:AttributeInfo` for details.
+
+To add a :class:`~tensorbay.label.label.Classification` label to one data:
+
+.. code:: python
+
+    >>> from tensorbay.dataset import Data
+    >>> data = Data(fileuri)
```

Comment:
the word `fileuri` is deprecated, also `fileuri` is difficult to understand here.
due to `Data` is used for local file only, it is better to use `local_path`